boltrouter/bolt_request.go

```diff
@@ -177,7 +177,7 @@ func (br *BoltRouter) DoBoltRequest(logger *zap.Logger, boltReq *BoltRequest) (*
 		} else if !StatusCodeIs2xx(resp.StatusCode) && br.config.Failover {
 			b, _ := io.ReadAll(resp.Body)
 			resp.Body.Close()
-			logger.Warn("bolt request failed", zap.Int("statusCode", resp.StatusCode), zap.String("body", string(b)))
+			logger.Error("bolt request failed", zap.Int("statusCode", resp.StatusCode), zap.String("body", string(b)))
```

Comment:
Should this really be an error, are we planning for all bolt request to always go through?

Reply:
We are not but this is to account for logging being restricted to errors only as this indeed is an error.